packages/vsx-registry/src/browser/style/index.css

```diff
@@ -147,6 +147,19 @@
 
 .theia-vsx-extension-editor .body {
     flex: 1;
+    margin: auto;
+}
+
+.theia-vsx-extension-editor .body h2 {
+    border-bottom: 1px solid #363636;
```

Comment:
Did you try that color is good in all themes?

Reply:
Also body is generated content, there is no guruantee that markdown will have header. I don't think that styling should rely on elements which are not under our control.

Reply:
> Also body is generated content, there is no guruantee that markdown will have header. I don't think that styling should rely on elements which are not under our control.

I don't quite understand this comment, I think @kaiyue0329 was simply updating `h2` elements (if they exist) to have their respective underlines. I also thought it might be useful to reuse our `markdown.css` for the preview of markdown files so the styling is consistent.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
> Did you try that color is good in all themes?

This color looks a bit dark under Light Themes. I have updated to a different color.

Reply:
> I don't quite understand this comment

ok, i thought it was drawing a separator for body against the header based on h2